README.md

```diff
@@ -40,4 +40,8 @@ from the README.md leaving only your finished README content for your project. -
 
 # Project Name
 
-<!-- TODO: Edit the project name heading and include a description of the project -->
\ No newline at end of file
+<!-- TODO: Edit the project name heading and include a description of the project -->
+
+# License
+$PROJECT_NAME is licensed under an [Apache 2.0 license](./LICENSE).
+The #PROJECT_NAME documentation is licensed under a [CC-BY-4.0 license](./LICENSE-docs). 
```

Comment:
I wonder if referencing two different licenses in the same github repo could cause problems?

As far as I can tell each existing repo either selects Apache 2.0 or CC-by-4.0 based on if it is a [code](https://github.com/kubernetes/kubernetes) or [docs](https://github.com/kubernetes/website) repo.

Could we do something like 

```suggestion
<!-- Select one of the below, based on if this is a code or docs repo. -->
$PROJECT_NAME is licensed under an [Apache 2.0 license](./LICENSE).
#PROJECT_NAME documentation is licensed under a [CC-BY-4.0 license](./LICENSE-docs). 
```

And we have two files in our template `LICENSE-apache` and `LICENSE-ccby40` and ask users to rename to `LICENSE` depending on the choice above?

Reply:
A lot of projects have their docs in the same repository as their code and don't break them out separately until the project is large or has complex docs. I think it would help to give them 3 options: code only repo, docs only repo, or combined repo.

Reply:
Options sounds good to me

Reply:
I'd rather have LICENSE-code and LICENSE-docs files, to remove any confusion

Reply:
It doesn't work with it.

Reply:
GitHub now supports and reports multiple licenses when a project has more than one. For details, see [Easily discover and navigate to multiple licenses in repositories](https://github.blog/changelog/2022-05-26-easily-discover-and-navigate-to-multiple-licenses-in-repositories/).

I see that this PR has been superseded by more recent changes (so maybe it should be closed or reworked), but I thought that I'd mention the new GitHub feature anyway.

Over on https://github.com/open-telemetry/opentelemetry.io (which is a **website** repo), we have:

- https://github.com/open-telemetry/opentelemetry.io/blob/main/LICENSE
- https://github.com/open-telemetry/opentelemetry.io/blob/main/LICENSE-CODE

GitHub links to them both just fine:

> <img width="963" alt="image" src="https://user-images.githubusercontent.com/4140793/176534358-86c9c0c7-6822-4a52-9529-bba5010fe6e7.png">

/cc @nate-double-u @caniszczyk

Reply:
OK, that definitely helps with that.